### ISSUE TITLE
Thread-local variable `:time_up_timers` is only available in the main thread 

### ISSUE THREAD

**Issue by janko** [testdouble/time_up#3]
Hi @searls, thanks for making this gem, I'm eager to try it out 🙂

I ran into an issue in a Rails app when trying to measure time in system tests. I wanted to run some timers around controller actions using before/after filters, to see how long they take, but I got this exception:

```
     NoMethodError:
       undefined method `[]' for nil:NilClass

     # /Users/jankomarohnic/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/time_up-0.0.6/lib/time_up.rb:9:in `timer'
     # /Users/jankomarohnic/.rbenv/versions/3.0.1/lib/ruby/gems/3.0.0/gems/time_up-0.0.6/lib/time_up.rb:14:in `start'
```

I'm assuming that's because when `time_up.rb` is loaded, it sets the `:time_up_timers` thread-local variable in the main thread, but when Capybara starts a new background thread, that one doesn't have the variable set, because it's only set when requiring `time_up`.

**Comment by janko** [comment]
Related to that, having the ability to measure time across multiple threads and being able to print out a merged summary would be a killer feature for me 👌🏻

**Comment by kylefox** [comment]
I'm running into the same error using `TimeUp.start(:my_timer) { some_code }` in a Rails controller (development environment). Not sure if it's related to threads though.

Thanks for the cool library, excited to use it 😀

**Comment by searls** [comment]
Great question! I did that at @al2o3cr's suggestion so I'm going to tag him in now to suggest a solution :P

**Comment by al2o3cr** [comment]
@searls Initializing inside of the module declaration: https://github.com/testdouble/time_up/blob/19ad87623c76f6e294acb69d12e77d5813a48693/lib/time_up.rb#L6 
results in only the thread that loaded `TimeUp` getting correctly initialized. 

Making `__timers` lazy-initialize is one way to solve this:
```
  def self.__timers
    Thread.current[:time_up_timers] ||= {}
  end
```

**Comment by searls** [comment]
Thanks Matt! Released 0.0.7 to do this; hopefully that fixes y'all
